packages/docs/pages/devguide/codestyle-and-linting.mdx

```diff
@@ -1,90 +1,62 @@
 import { Callout } from 'nextra-theme-docs'
 
-# Code style and linting
-
-For enforcing code style and utilizing static code analyzing we use [Prettier](https://prettier.io/) and [ESLint](https://eslint.org/).
-
-## Prettier configuration
-
-You can always execute the commands `pnpm format:(write|check)` to apply the Prettier formatting settings for all project files inside the
-`src`-directory (`write` actually writes the formattings to the files whereas `check` only does a dry-run).
-
-### Rules
-
 <Callout type="warning" emoji="⚠️">
-  Todo: Check if the config can be imported from the prettier config directly
+  Some sections are still drafts and marked with a 📝 symbol. Feel free to
+  contribute 🎉.
+  <ul>
+    <li>- add Prettier rules explanation</li>
+    <li>- check intro paragraph if the order of context makes sense</li>
+  </ul>
```

Comment:
Have you created an issue for this?

Reply:
Good point --> https://github.com/Frachtwerk/essencium-frontend/issues/394